test/typescript/t.test.ts

```diff
@@ -16,9 +16,9 @@ function overloadedUsage(t: TFunction) {
 function returnCasts(t: TFunction) {
   const s: string = t('friend'); // same as <string>
   const s2: string = t`friend`;
-  const o: object = t<object>('friend');
-  const sa: string[] = t<string[]>('friend');
-  const oa: object[] = t<object[]>('friend');
+  const o: object = t<string, object>('friend');
+  const sa: string[] = t<string, string[]>('friend');
+  const oa: object[] = t<string, object[]>('friend');
```

Comment:
I don't disapprove of this, but it should be noted that this technically is a type-breaking change.

Reply:
yeah, I agree. Just want to reinforce that it should be a `Minor` release, not only a `Patch` release

Reply:
By semver requirements, breaking would be `major`.  I don't have a strong preference one way or the other, I just worry about those that do expect strict adherence to semver.